src/renderer/renderScene.ts

```diff
@@ -205,32 +203,60 @@ const renderLinearPointHandles = (
   if (!centerPoint) {
     return;
   }
+  const { POINT_HANDLE_SIZE } = LinearElementEditor;
+
   points.forEach((point, idx) => {
     const isSelected =
       !!appState.editingLinearElement?.selectedPointsIndices?.includes(idx);
-    renderSingleLinearPoint(context, appState, renderConfig, point, isSelected);
+    const radius = appState.editingLinearElement
+      ? POINT_HANDLE_SIZE
+      : POINT_HANDLE_SIZE / 2;
```

Comment:
Can be kept outside — why calculate this inside the loop, and on two separate places?

Reply:
ofcourse! I dunno why I was doing that 🤦‍♀️